pandas/tests/arithmetic/test_object.py

```diff
@@ -281,6 +282,8 @@ def test_add(self):
 
     def test_sub_fail(self):
         index = tm.makeStringIndex(100)
+
+        # TODO: Make raised error message more informative and test
```

Comment:
Any reason for the TODO here instead of adding the message?

Reply:
@WillAyd . I was not sure what a good error message for this function would be.  Does the error message added in my previous commit work?